lib/tether/ops_darwin.go

```diff
@@ -20,12 +20,11 @@ import (
 	"fmt"
```

Comment:
are changes very different from ops_linux? I would consider keeping platform independent logic in one file